spec.bs

```diff
@@ -1775,7 +1787,7 @@ An auction config is a [=struct=] with the following items:
   [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] restrict the number of
   bidding interest groups for a particular buyer that can participate in an auction.
 : <dfn>all buyers group limit</dfn>
-:: A {{unsigned short}}.
+:: An {{unsigned short}}. Defaulting to 65535.
   Limit on the number of bidding interest groups for all buyers without a limit specified in
   [=auction config/per buyer group limits=].
 : <dfn>per buyer priority signals</dfn>

```

Comment:
Maybe this should link to a section about prioritization?

Reply:
I don't think there is a specific section that discusses prioritization unfortunately.  We could link to the huge `runAdAuction()` section but I don't think that would be particularly helpful.